src/Authentication/Actions/Email2FA.php

```diff
@@ -70,13 +75,17 @@ public function handle(IncomingRequest $request)
 
         // Send the user an email with the code
         helper('email');
-        $email = emailer();
-        $email->setFrom(setting('Email.fromEmail'), setting('Email.fromName') ?? '')
+        $emailer = emailer();
```

Comment:
Is the intermediate `$emailer` variable necessary?

Reply:
Removed.